src/ServicePulse.Host/vue/src/components/monitoring/EndpointDetails.vue

```diff
@@ -466,18 +466,18 @@ onMounted(() => {
               <!--ProcessingTime and Critical Time-->
               <div class="col-xs-4 no-side-padding list-section graph-area graph-critical-processing-times">
                 <!-- large graph -->
-                <D3LargeGraph
+                <LargeGraph
                   v-if="endpoint.metricDetails.metrics.criticalTime"
                   :isdurationgraph="true"
                   :firstdataseries="endpoint.metricDetails.metrics.criticalTime"
                   :seconddataseries="endpoint.metricDetails.metrics.processingTime"
                   :minimumyaxis="largeGraphsMinimumYAxis.processingCritical"
-                  :firstseriescolor="'#2700CB'"
-                  :firstseriesfillcolor="'#C4BCE5'"
-                  :secondseriescolor="'#258135'"
-                  :secondseriesfillcolor="'#BEE6C5'"
+                  :firstseriestype="'critical-time'"
+                  :firstseriesavgcolour="'#2700CB'"
```

Comment:
Why pass in these colors and not make apply the type of graph as a class to the tooltip and style them with css?

Reply:
Good idea. We can do that in another PR, along with fixing using "color" instead of "colour" for consistency.